rest/client.go

```diff
@@ -17,6 +17,7 @@ type GetComponentsFilter struct {
 }
 
 type Client interface {
+	GetComponent(context.Context, uuid.UUID) (models.GetComponentResponse, error)
```

Comment:
We should return the models.Component here instead of the response

Reply:
Cool will fix this